scripts/pre_commit_linter.py

```diff
@@ -44,12 +44,11 @@
 
 __author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
 
+import sys
+import time
 import argparse
 import os
 import subprocess
-import sys
-import time
```

Comment:
yes, I am using Python 2.7. I noticed that using the Pylint installed with Oppia (1.5.2) caused this error, while the Pylint that is on my local system does not throw this error.